pallets/xcm-app/src/lib.rs

```diff
@@ -275,6 +285,7 @@ pub mod pallet {
             asset_id: AssetId,
             multiasset: xcm::v3::AssetId,
             asset_kind: bridge_types::types::AssetKind,
+            min_amount: u128,
```

Comment:
Do you have special type for balance here?

Reply:
no, the belance is mapped properly on sora2-network